src/status_im2/contexts/onboarding/identifiers/profile_card/style.cljs

```diff
@@ -0,0 +1,33 @@
+(ns status-im2.contexts.onboarding.identifiers.profile-card.style
+  (:require [react-native.reanimated :as reanimated]
+            [quo2.components.avatars.user-avatar.style :as avatar-style]))
+
+(defn card-container
+  [background-style]
+  {:style [(reanimated/apply-animations-to-style
+            {}
+            {:padding-horizontal 12
+             :padding-top        12
+             :padding-bottom     12
+             :flex               1
+             :border-radius      16})
+           background-style]})
```

Comment:
Hi @websky112, Thank you for great work :tada: 

Why we are using animated style, when animated values map is empty?
Should we get only derived value of backgroundColor instead of style, and use that?

Reply:
I believe this should be:
```
 {:style [{:padding-horizontal 12
             :padding-top        12
             :padding-bottom     12
             :flex               1
             :border-radius      16}
           background-style]})
     ```

Reply:
Done using derivedValue instead of style. 

Concern is that normal style doesn't apply when writing code as @J-Son89 showed above. I'm sure there will be cases where we need to use useAnimatedStyle hook and I hope this problem would be resolved.
@J-Son89 @Parveshdhull 
Thanks